### PR TITLE
New Organisation Identifier: XI-IATI-SPR

### PR DESCRIPTION
**Pull request by andreaszenasidi**
Approved IATI Connect consultation: https://iaticonnect.org/group/standard-management-consultations-0/discussion/new-organisation-identifier-xi-iati-spr

### CODE REVIEW

**Comment by nosvalds** [comment]
I didn't think we maintained this codelist anymore? 

https://iatistandard.org/en/iati-standard/203/codelists/organisationidentifier/

> As of 1.04 this list is no longer maintained. http://support.iatistandard.org/entries/28497976-Retire-the-Organisation-Identifier-codelist#view-post-25368673

**Review by PetyaKangalova — APPROVED** [review]
Looks good as per Connect post: https://iaticonnect.org/group/standard-management-consultations-0/discussion/new-organisation-identifier-xi-iati-spr

@nosvalds  could you also review?

It is a bit confusing but we still maintain the XI-IATI organisation identifier list, where this addition is being made(https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/master/xml/IATIOrganisationIdentifier.xml)

The organisation identifier codelist (https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/master/xml/OrganisationIdentifier.xml) is indeed deprecated :)

File: xml/IATIOrganisationIdentifier.xml [hunk 1]
```diff
@@ -334,6 +334,16 @@
...
         </codelist-item>
+        <codelist-item>
+            <code>XI-IATI-SPR</code>
+            <name>
+                <narrative>Synergie des Femmes pour la Paix et la Réconciliation des Peuples des Grands Lacs d'Afrique (SPR)</narrative>
+            </name>
+            <description>
+                <narrative>Synergie des Femmes pour la Paix et la Reconciliation des Peuples des Grands Lacs d’Afrique is a network of 37 human rights organisations in the Great Lakes region of Africa (Eastern DR Congo, Rwanda, and Burundi). The network was created in 2005 as a sub-regional initiative of the Pax Christi Great Lakes Network to use the potential of women in the region to build peace and promote of women's rights, leadership, and empowerment.</narrative>
+            </description>
+            <url>http://www.sprgrandslacs.org/</url>
```
Comment (nosvalds) [inline xml/IATIOrganisationIdentifier.xml:345]:
```suggestion
            <url>https://www.sprgrandslacs.org/</url>
```

**Review by nosvalds — APPROVED** [review]
Just one small update requested to make the url `https` instead of `http`.

Can you also add a Trello ticket for pushing this to the SSOT on the Website?

**Comment by andreaszenasidi** [comment]
@nosvalds new Trello card was created.

**Pull request merged**